docs/querying/sql-functions.md

```diff
@@ -1306,35 +1307,118 @@ Returns the following:
 
 ## IPV4_MATCH
 
-`IPV4_MATCH(address, subnet)`
+Returns true if the IPv4 `address` belongs to the `subnet` literal, otherwise returns false.
 
-**Function type:** [Scalar, IP address](sql-scalar.md#ip-address-functions)
+* **Syntax:** `IPV4_MATCH(address, subnet)`
+* **Function type:** Scalar, IP address
 
-Returns true if the IPv4 `address` belongs to the `subnet` literal, else false.
+<details><summary>Example</summary>
+
+The following example returns true if the IPv4 address in the `forward_for` column from the `kttm` datasource belongs to the subnet `181.13.41.0/24`.
+
+```sql
+SELECT 
+  "forwarded_for" AS "ipv4_address",
+  IPV4_MATCH("forwarded_for", '181.13.41.0/24') AS "belongs_in_subnet"
+FROM "kttm"
+LIMIT 2
+```
+
+Returns the following:
+
+| `ipv4_address` | `belongs_in_subnet`|
+| -- | -- | 
+| `181.13.41.82` | `true`|
+| `177.242.100.0` | `false`|
+
+</details>
 
-## IPV4_PARSE
 
-`IPV4_PARSE(address)`
+[Learn more](sql-scalar.md#ip-address-functions)
 
-**Function type:** [Scalar, IP address](sql-scalar.md#ip-address-functions)
+
+## IPV4_PARSE
 
 Parses `address` into an IPv4 address stored as an integer.
```

Comment:
this description seems wrong to me based upon the example. I would say:
> Parses an  IPv4 `address` as an integer.

Please verify.